lib/PluginBase.js

```diff
@@ -40,8 +33,8 @@ class PluginBase {
     /**
      * Method for Plugin developer to initialize his Plugin
      *
-     * @param pluginConfig {object} plugin configuration from config files
-     * @param callback {function} callback when done, signature "(err, initSuccessful)". On err or initSuccessful===false the plugin instance will be discarded
+     * @param {Record<string, any>} pluginConfig plugin configuration from config files
+     * @param {import("@iobroker/plugin-base/types").InitCallback} callback Will be called when done. On err or `initSuccessful === false` the plugin instance will be discarded.
```

Comment:
A bit ugly, but this way implementations can just copy the method signature including JSDoc comments and have full type support.